src/components/Grades/AnimatedEmoji.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from "react";
+import { View, Text } from "react-native";
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+  withSpring,
+  withSequence,
+  Easing,
+} from "react-native-reanimated";
+
+interface AnimatedEmojiProps {
+  initialScale?: number;
+  size?: number; // Taille de la police
+}
+
+const AnimatedEmoji: React.FC<AnimatedEmojiProps> = ({ initialScale = 1, size = 20 }) => {
+  const scale = useSharedValue(initialScale);
+  const opacity = useSharedValue(1);
+  const emojis = ["😍", "🙄", "😭", "🥳", "😱", "😳", "🤓", "🤡", "🤯", "😨", "🤔", "🫠"];
+  const [currentEmoji, setCurrentEmoji] = React.useState(emojis[0]);

```

Comment:
Autant faire l'import directement :

```suggestion
  const [currentEmoji, setCurrentEmoji] = useState(emojis[0]);
```

Reply:
C'est possible de choisir ses propres emojis plutôt que via une courte liste?

Sinon aussi rajouter ces emojis 😫🤫🥱😈🤐😇😏🤠